### DISCUSSION TITLE
headless. cannot currently show the desktop

### DISCUSSION THREAD

**Discussion by gildas-declercq** [bareboat-necessities/lysmarine_gen#346] [Q&A] [answered]
Hi all,
I will most of the time connect with VNC on my boat rpi BBN.
it is ok when it starts connected to a screen
but if started with no screen I got the "cannot currently show the desktop"

I did set the VNC resolution with rapi-config

I also tried vncserver-virtual
which result as
"New desktop is lysmarine:1 (192.168.107.108:1)"

but with no success
 if I plug back the screen it does not show anything.

any help or advise is welcome

thanx
Gildas

**Comment by mgrouch** [comment, marked as answer]
set

hdmi_force_hotplug=1

in /boot/config.txt

**Reply by gildas-declercq** [reply]
perfect. sorry if I missed that in the doc.  I also did a search on the net but nothing was leading me to the right answer
